tests/neg/i13815.check

```diff
@@ -3,7 +3,14 @@
 1 |val box = Box.fromProduct((1, "a"))  // error
   |          ^^^^^^^^^^^^^^^
   |          value fromProduct is not a member of object Box
--- [E008] Not Found Error: tests/neg/i13815/Test_2.scala:2:26 ----------------------------------------------------------
+-- [E007] Type Mismatch Error: tests/neg/i13815/Test_2.scala:2:46 ------------------------------------------------------
 2 |val circleOrdinal = Shape.ordinal(Shape.Circle(r = 23)) // error
-  |                    ^^^^^^^^^^^^^
-  |                    value ordinal is not a member of object Shape
+  |                                  ^^^^^^^^^^^^^^^^^^^^
+  |                                  Found:    Shape.Circle
```

Comment:
this seems strange that making `MirroredMonoType` invisible now makes `ordinal` visible again